### ISSUE TITLE
Accessibility: jupyter home page

### ISSUE THREAD

**Issue by Abolade017** [jupyterhub/outreachy#83]
### What page is this for?

https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1274666215

### WAVE accessibility report

My name is Adeola Tawakalitu Adenike. After accessing the `jupyter.org` website with WAVE tool, the evaluation to the website accessibility is the image below:

![jupytercon page](https://user-images.githubusercontent.com/70269933/195077971-eac08b3b-0d7d-4c29-ad6b-4872067e8a25.JPG)

The following is the detailed evaluation done on the website accessibility:

-  **Contrast error (7) :** There is a very low contrast between text and background colors. We can fix this error by increasing the contrast between the foreground (text) color and the background color. Large text (larger than 18 point or 14 point bold) does not require as much contrast as smaller text.
- **A nearby image has the same alternative text (2) :** We can fix this alert by ensuring that the alternative text for each image or image button is appropriate while removing unnecessary redundancy. If the content of the image is already conveyed elsewhere (through text or the alternative text of a nearby image) or if the image does not convey content, the image may generally be given empty/null alternative text (alt=""). Image buttons always convey a specific function, and thus cannot be given null alternative text. 
- **Missing `<h1>` first level element (1) :** Headings facilitate page navigation for users of many assistive technologies. They also provide semantic and visual meaning and structure to the document. A first level heading (`<h1>`) should be present on nearly all pages. It should contain the most important heading on the page (generally the document title). If the page presents a main heading, place it within an `<h1>` element. Add other sub-headings as necessary.
- **Possible heading (2) :** Text appear to be a heading but it is not a heading element. Heading elements (`<h1>-<h6>`) provide important document structure, outlines, and navigation functionality to assistive technology users. If heading text is not a true heading, this information and functionality will not be available for that text. If the paragraph is a section heading, use a heading element instead (`<h1>-<h6>`).
- **Images having alternative text that is not equivalent to the images function or content (20) :** Alternative text presents the content or function of an image to screen reader users or in other situations where images cannot be seen or are unavailable. it can be fix by ensuring that the alternative text conveys the content and function of the image accurately and succinctly. The alt attribute should be equivalent, accurate, and succinct.
- **Link images with alternative text (30) :** Alternative text is present for an image that is within a link. This can be fixed by ensuring that the alternative text presents the content of the image and/or the function of the link. If the full content and function of the link is presented in text within the link (an image and a text caption both within the same link, for example), then the image should generally be given empty/null alternative text (`alt=""') to avoid redundancy. 
- **Language (1) :** Language of the document or a page element is identified. Identifying the language of the page or portion of page (if different from the page itself) allows screen readers to read the content appropriately. It can be fixed by ensuring the language is properly identified for the page (e.g., `<html lang="en">`). If content within the page is in a language different than the page's language, identify it using a valid `lang` attribute value (e.g., `<p lang="fr">`).
- **Heading level (19) :** This comprises of seven heading level two (`<h2>`) and twelve heading level three (`<h3>`) This can be fixed by ensuring that the text in question is truly a heading and that it is structured correctly in the page outline.
- **Unordered list (5) :** An unordered (bulleted) list (`<ul>` element) is present. it can be fixed by ensuring that an unordered (bulleted) list is appropriate for the context. If list items are sequential or numbered, an ordered list (`<ol>`) is likely more appropriate. 
- **Navigation (1) :** A `<nav>` element or navigation landmark is present. The navigation identifies a section of navigation links and can facilitate page semantics and navigation. it can be fixed by ensuring the element defines page navigation. Multiple navigation elements on one page can be differentiated with `ARIA` labels.
- **Footer (1) :** A `<footer>` element or `contentinfo` landmark is present.Footers identify a footer for the page or a page section. It typically identifies authorship, related links, copyright date, or other footer content. Footers facilitate page semantics and navigation. it can be fixed by ensuring  the element surrounds and defines page or page section footer content.

### The HTML element to be changed

```html
1. The document structure should be changed, because all page content should contain landmark. the structure is <body> <section> <header> <article> and <footer>.
2.The <p> element in <div class="con-program">
<p class="dropshadow">Cité des Science, Paris, France</p>
<p class="dropshadow">CFP open Oct 31st</p>
</div> should be changed.
3. The change the background color used for <div class="button-container">
 <a class="orange-button " href="/try">Try it in your browser</a>
<a class="orange-button install-button" href="/install">Install JupyterLab</a>
</div> which is:
 homepage-section .orange-button {
  font-size: 1.2em;
  background-color: #e46e2e;
  border-radius: 2px;
  color: #fff;
  text-shadow: 0px 0px 4px #000;
  padding: 0.75em 1.25em;
  margin: 0 0.5em 0 0;
  text-align: center;
  text-decoration: none;
}
4. I will change the alternative text of the images that are placed beside each other.
<div class="jumbotron"><div class="col-md-12"><div class="jumbotron-image-container"><span>
Alternative text<img class="img-responsive fade-in one" src="/assets/homepage/circle1.svg" alt="circle of programming language icons" loading="eager" width="677" height="286">
</span></div><div class="jumbotron-image-container"><span>
A nearby image has the same alternative text<img class="img-responsive fade-in two" src="/assets/homepage/circle2.svg" alt="circle of programming language icons" loading="eager" width="677" height="286">
</span></div><div class="jumbotron-image-container"><span>
A nearby image has the same alternative text<img class="img-responsive fade-in three" src="/assets/homepage/circle3.svg" alt="circle of programming language icons" loading="eager" width="677" height="286">
</span></div>
```

### Your proposed HTML (or CSS), after change

```html
1. i will fix this issue by giving the document structure the appropriate structure <body><header></header><main></main><footer></footer></body>
2. I will change the <p> element in to <h1> and <h2> element because the document should only have a single <h1> element through out the page. The code is this:
 <div class="con-program">
<h1 class="dropshadow">Cité des Science, Paris, France</h1>
<h2 class="dropshadow">CFP open Oct 31st</h2>
</div> 
3.I will change the background-color and also add opacity to it 
.homepage-section .orange-button{
background-color:#f85801;
 opacity:0.8;
}
4. I will give the alt attribute value different name. e.g 
<div class="jumbotron"><div class="col-md-12"><div class="jumbotron-image-container"><span>
Alternative text<img class="img-responsive fade-in one" src="/assets/homepage/circle1.svg" alt="circle of programming language icons" loading="eager" width="677" height="286">
</span></div><div class="jumbotron-image-container"><span>
A nearby image has the same alternative text<img class="img-responsive fade-in two" src="/assets/homepage/circle2.svg" alt="first circle of programming language icons" loading="eager" width="677" height="286">
</span></div><div class="jumbotron-image-container"><span>
A nearby image has the same alternative text<img class="img-responsive fade-in three" src="/assets/homepage/circle3.svg" alt="second circle of programming language icons" loading="eager" width="677" height="286">
</span></div>
4.
```

**Comment by minrk** [comment]
Thank you for the suggestions!
